nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java

```diff
@@ -84,6 +89,72 @@
             .allowableValues("0.9.1")
             .defaultValue("0.9.1")
             .build();
+    public static final PropertyDescriptor USE_SSL_PROTOCOL = new PropertyDescriptor.Builder()
+            .name("Use SSL protocol")
+            .description("Indicates whether or not SSL protocol should be used")
+            .allowableValues("true", "false")
+            .defaultValue("false")
+            .required(true)
+            .build();
+    public static final PropertyDescriptor VALIDATE_CERTIFICATES = new PropertyDescriptor.Builder()
```

Comment:
@alopresto In fact I added this possibility because this is a use case described in [the RabbitMQ documentation](https://www.rabbitmq.com/ssl.html), and I assumed this is a possible scenario. Regarding the default value, I definitely agree with you, my bad.